src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -229,6 +235,41 @@ module approveKeyVaultPrivateEndpointConnections 'keyVaultEndpoints.bicep' = {
   }
 }
 
+resource dataExplorerManagedPrivateEndpoint 'Microsoft.DataFactory/factories/managedVirtualNetworks/managedPrivateEndpoints@2018-06-01' = if (deployDataExplorer) {
+  name: dataExplorerName
```

Comment:
Shouldn't we use "hubDataExplorer" like in the linked service? That will be simpler to debug because we always know what the names are.

Reply:
Done